index.html

```diff
@@ -0,0 +1,234 @@
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@200;400&family=Poppins:wght@100;500&display=swap" rel="stylesheet">
+</head>
+<body>
+<main>
+    <header>
+        <nav>
+            <ul class="logo" >
+                <li><a href=""> <h3 class="gakii"> Gakii Beryl</h3></a></li>
+                <li><a href=""><img class="menu" src="images/menu.svg" alt="Menu Icon" /></a></li>
+            </ul>
+        </nav>
+    </header>
+    <nav class="headline">
+        <div class="about">
+            <h1>Hey There.<br />I'm Gakii </h1>
+            <h2>I'm a software Developer</h2>
+            <p>
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and have a
+                project you need coded, don't hestiate to contact me.
+            </p>
+            <ul class="social_media">
+                <li><a href="https://github.com/berylcl"><img src="images/github.png" alt="git" width="18px" height="20px"></a></li>
+                <li><a href="https://www.linkedin.com/in/callme-claudette-83a796230/"><img src="images/Icon%20-%20Linkedin.png" alt="linkedin" width="24px" height="24px"></a></li>
+                <li><a href="#"><img src="images/vic.png" alt="vic" width="14.17px" height="20px"></a></li>
+                <li><a href="https://twitter.com/omg_its_beryl"><img src="images/twitter.png" alt="twitter" width="20px" height="20px"></a></li>
+                <li><a href="#"><img src="images/medium.png" alt="" width="20ox" height="16px"></a></li>
+            </ul>
+        </div>
+    </nav>
+<div class="recent-works">
+    <section class="headline2">
+        <div class="container1">
+            <div class="flex"><h1 class="recent">My Recent works</h1><hr></div>
+            <div class=" contents">
+                <img src="images/multi-post.png" width="327" height="237">
+
+                <h2 class="post1">Multi-post Stories</h2>
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <div class="languages">
+                <button type="button" class="css">css</button>
+                <button type="button"class="html">html</button>
+                <button type="button" class="bootstrap">bootstrap</button>
+                <button type="button" class="Ruby">Ruby</button>
+                <button type="button"class="btn color">See Project</button>
+            </div>
+
+        </div>
+    </section>
+        <div class="card">
+            <div class="printing-data">
+                <div class="right-block">
+                    <div class="printing-title">
+                        <h1>Professional Art <br>Printing Data</h1>
```

Comment:
- [ ] According to best semantics and SEO principles, there should be only one `h1` tag on your entire webpage. so kindly consider taking other `h` tags here and in other cards as well where you are taking `h1` tag.